scripts/add-javascript-rules.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Might be worth making this compatible for windows machine because I am pretty sure we (dotnet team) will be the one updating the website for SonarLint for VS. WDYT?